app/static/src/scss/base.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 
-@media screen and (width <=768px) {
+@media screen and (width <=810px) {
```

Comment:
is there a reason we're changing the mobile threshold?

Reply:
This should account for both tablet & mobile sizing. The summary-list is a bit more fluid when it comes to different widths

Reply:
ah I see, thanks for clarifying